src/sentry/api/endpoints/group_hashes.py

```diff
@@ -42,3 +45,26 @@ def get(self, request, group):
             order_by='id',
             on_results=lambda x: serialize(x, request.user),
         )
+
+    def delete(self, request, group):
+        id_list = request.GET.getlist('id')
+        if id_list is None:
+            return Response()
+
+        hash_list = GroupHash.objects.filter(
```

Comment:
Ah, got it. I was previously thinking about adding a `status` field to `GroupHash` to prevent users from attempting to unmerge the same hash multiple times but hadn't done that in this change. I think it'd make sense to update to reflect the status when that change is made.